processors/auditlog/auditlogProcessor.go

```diff
@@ -0,0 +1,30 @@
+package auditlog
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"strings"
+)
+
+type AuditLogProcessor struct {
+	AuditLogChan chan string
+}
+
+func (a *AuditLogProcessor) Process(ctx context.Context, r io.Reader, readBytes *bytes.Buffer, buf []byte) (int, error) {
```

Comment:
this func reads in bytes from the namedpipe ingester and pushes log entries to the channel for every newline.